### ISSUE TITLE
update code list with positive attribute

### ISSUE THREAD

**Issue by larsbuntemeyer** [remo-rcm/tables#51]
Here is the positive attributes from the old mapping table:
```

&parameter project_mip_table=sem name=hfls        cmor_name=hfls     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=sem name=hfss        cmor_name=hfss     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=sem name=rlds        cmor_name=rlds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=sem name=rlus        cmor_name=rlus     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=sem name=rlut        cmor_name=rlut     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=sem name=rsds        cmor_name=rsds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=sem name=rsdt        cmor_name=rsdt     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=sem name=rsus        cmor_name=rsus     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=sem name=rsut        cmor_name=rsut     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=sem name=tauu        cmor_name=tauu     p="d"   units="Pa"           cell_methods="m" /
&parameter project_mip_table=sem name=tauv        cmor_name=tauv     p="d"   units="Pa"           cell_methods="m" /
&parameter project_mip_table=mon name=hfls        cmor_name=hfls     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=mon name=hfss        cmor_name=hfss     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=mon name=rlds        cmor_name=rlds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=mon name=rlus        cmor_name=rlus     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=mon name=rlut        cmor_name=rlut     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=mon name=rsds        cmor_name=rsds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=mon name=rsdt        cmor_name=rsdt     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=mon name=rsus        cmor_name=rsus     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=mon name=rsut        cmor_name=rsut     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=mon name=tauu        cmor_name=tauu     p="d"   units="Pa"           cell_methods="m" /
&parameter project_mip_table=mon name=tauv        cmor_name=tauv     p="d"   units="Pa"           cell_methods="m" /
&parameter project_mip_table=day name=hfls        cmor_name=hfls     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=day name=hfss        cmor_name=hfss     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=day name=rlds        cmor_name=rlds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=day name=rlus        cmor_name=rlus     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=day name=rlut        cmor_name=rlut     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=day name=rsds        cmor_name=rsds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=day name=rsdt        cmor_name=rsdt     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=day name=rsus        cmor_name=rsus     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=day name=rsut        cmor_name=rsut     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=day name=tauu        cmor_name=tauu     p="d"   units="Pa"           cell_methods="m" /
&parameter project_mip_table=day name=tauv        cmor_name=tauv     p="d"   units="Pa"           cell_methods="m" /
&parameter project_mip_table=6hr name=rlds        cmor_name=rlds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=6hr name=rsds        cmor_name=rsds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=6hr name=rsus        cmor_name=rsus     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=3hr name=hfss        cmor_name=hfss     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=3hr name=rlds        cmor_name=rlds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=3hr name=rsds        cmor_name=rsds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=3hr name=rsus        cmor_name=rsus     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=3hr name=rlus        cmor_name=rlus     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=3hr name=hfls        cmor_name=hfls     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=1hr name=rlds        cmor_name=rlds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=1hr name=rsds        cmor_name=rsds     p="d"   units="W m-2"        cell_methods="m" /
&parameter project_mip_table=1hr name=rsus        cmor_name=rsus     p="d"   units="W m-2"        cell_methods="m" /
```

**Comment by larsbuntemeyer** [comment]
The have to be `positive=down`:
```
['hfls',
 'hfss',
 'rlds',
 'rlus',
 'rlut',
 'rsds',
 'rsdt',
 'rsus',
 'rsut',
 'tauu',
 'tauv',
 'hfls',
 'hfss',
 'rlds',
 'rlus',
 'rlut',
 'rsds',
 'rsdt',
 'rsus',
 'rsut',
 'tauu',
 'tauv',
 'hfls',
 'hfss',
 'rlds',
 'rlus',
 'rlut',
 'rsds',
 'rsdt',
 'rsus',
 'rsut',
 'tauu',
 'tauv',
 'rlds',
 'rsds',
 'rsus',
 'hfss',
 'rlds',
 'rsds',
 'rsus',
 'rlus',
 'hfls',
 'rlds',
 'rsds',
 'rsus']
```

**Comment by larsbuntemeyer** [comment]
closed by #52
